test/common.py

```diff
@@ -256,17 +255,12 @@ def run_train_translate(train_params: str,
     :param dev_source_factor_paths: Optional list of paths to dev source factor files.
     :param test_source_factor_paths: Optional list of paths to test source factor files.
     :param use_prepared_data: Whether to use the prepared data functionality.
+    :param use_target_constraints: Whether to use lexical constraints in the second translation pass.
```

Comment:
Not clear what the "second translation pass" means here. It sound like this is related to translate_params_equiv (which is not).